dissect/target/plugins/os/unix/log/auth.py

```diff
@@ -279,9 +276,19 @@ def build_record(self, ts: datetime, source: Path, line: str) -> TargetRecordDes
 
         for key, value in self._parse_additional_fields(record_values["service"], line).items():
             record_type = "string"
+
             if isinstance(value, int):
                 record_type = "varint"
 
+            elif isinstance(value, list):
+                record_type = "string[]"
```

Comment:
Won't this catch before `remote_ips` and `host_ips` since the value will also be a list?